src/IntegrationTests/IntegrationTests/Fixtures/OpenTelemetryUsageApplication.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml;
+
+namespace IntegrationTests.Fixtures
+{
+    public class OpenTelemetryUsageApplication : BaseApplication
+    {
+        private const string TestPackageName = "OpenTelemetry.Exporter.NewRelic";
+
+        private readonly string TestPackageVersion;
+
+        private readonly string ExporterNugetPackageOutputPath;
+
+        private readonly string TelemetrySdkNugetPackageOutputPath;
+
+        public string TestSampleApplicationDirectory { get; }
+
+        public string SolutionConfiguration { get; }
+
+        public string ApplicationOutputPath { get; private set; }
+
+        public OpenTelemetryUsageApplication(string applicationName, string[] serviceNames) : base(applicationName, serviceNames)
+        {
+            SolutionConfiguration = "Release";
+#if DEBUG
+            SolutionConfiguration = "Debug";
+#endif
+
+            ExporterNugetPackageOutputPath = Path.GetFullPath(Path.Combine(SrcDirectoryPath, $@"NewRelic.Telemetry\bin\{SolutionConfiguration}"));
+            TelemetrySdkNugetPackageOutputPath = Path.GetFullPath(Path.Combine(SrcDirectoryPath, $@"OpenTelemetry.Exporter.NewRelic\bin\{SolutionConfiguration}"));
+            
+            NugetSources.Add(ExporterNugetPackageOutputPath);
+            NugetSources.Add(TelemetrySdkNugetPackageOutputPath);
+
+            TestPackageVersion = GetNugetPackageVersion(ExporterNugetPackageOutputPath);
```

Comment:
I'm getting an empty string here. Framework-specific subdirectory is required ("netstandard2.0").
As a result, I think the package is getting pulled from nuget.org, not our build dir package source.
I'm thinking we may want to have separate RestoreNuGetPackage calls for the Exporter/SDK packages that explicitly use the build dir package sources.

Reply:
I have to take a look at where the nuget packages are put on your machine. For me, they are put outside of the framework specific folders, which makes sense because the nuget packages are packed in the way that are muti-framework targeted.

I don't think we can separate calls to restore Exporter and SDK pakages because the Exporter depends on the SDK.

Reply:
Not sure why the build dir package location would be different on my machine vs. yours vs. server. Let's get @jfeingold to try it on his machine.